### ISSUE TITLE
Reduce import to a single statement.

### ISSUE THREAD

**Issue by oscbyspro** [oscbyspro/DiffableTextViews#53]
While splitting the project into multiple modules keeps things organized

```swift
import DiffableTextViews
```

does effectively nothing. Instead, one must import it alongside a style

```swift
import DiffableTextViews
import NumericTextStyles
import PatternTextStyles
```

I think it would be worthwhile investigating the use of `@_exported import`.

**Comment by oscbyspro** [comment]
It is now sufficient to `import DiffableTextViews` in the `dev_milestone_v3.0.0` branch. It is also possible to import every other target, which is an unintended (minor) consequence of using `@_exported`.

**Comment by oscbyspro** [comment]
Solved as a consequence of #54.
